auth/rhoas_auth.py

```diff
@@ -1,12 +1,13 @@
 from keycloak import KeycloakOpenID
-from os import environ;
+from os import environ
+from auth.constants import DEFAULT_AUTH_URL, DEFAULT_CLIENT_ID, REALM_NAME;
 
 def get_access_token():
     offlineToken = environ.get('OFFLINE_TOKEN')
     # Configure client
-    keycloak_openid = KeycloakOpenID(server_url="https://sso.redhat.com/auth/",
-                                 client_id="cloud-services",
-                                 realm_name="redhat-external")
+    keycloak_openid = KeycloakOpenID(server_url=DEFAULT_AUTH_URL,
+                                 client_id=DEFAULT_CLIENT_ID,
+                                 realm_name=REALM_NAME)
```

Comment:
I am on the VPN but even so, when I run an example, I get this error:
```python
keycloak.exceptions.KeycloakPostError: 404: b'{"error":"Realm does not exist"}'
```

@wtrocki, have you seen this before?

Reply:
You have 2 PRs. Should other one be closed?

With config like this i do not see errors:
```
    keycloak_openid = KeycloakOpenID(server_url="https://sso.redhat.com/auth/",
                                 client_id="cloud-services",
                                 realm_name="redhat-external")
                                 
                             ```